shared/Cargo.toml

```diff
@@ -5,7 +5,19 @@ version = "0.1.0"
 edition = "2021"
 license = "AGPL-3.0"
 
+[lib]
+crate-type = ["cdylib", "rlib"]
+doctest = false
+
 [features]
+default = ["pg15"]
+pg11 = ["pgrx/pg11", "pgrx-tests/pg11" ]
```

Comment:
you can remove this one, pg11 is no longer supported by pgrx, I removed it everywhere else